extensions/amp-form/0.1/amp-form.js

```diff
@@ -769,12 +773,19 @@ export class AmpForm {
   }
 
   /**
-   * Transition the form to the submit success state.
+   * Transition the form to the submit-success or submit-error state depending on the response status.
    * @param {!JsonObject} response
    * @return {!Promise}
    * @private
    */
-  handleSsrTemplateSuccess_(response) {
+  handleSsrTemplateResponse_(response) {
+    if (
+      response.init &&
+      response.init.status &&
```

Comment:
These two lines looks confusing, say if response.init is not present, should we consider that as a failure or success?

I feel you should use 
!response.init || response.init.status || xxx
but I might be wrong

Reply:
You mean `!response.init || !response.init.status || response.init.status > threshold` ? 

Discussed offline abt this change, resolution:
If there's an assertion in the runtime that response.init is non-nullable, we should go with TC's suggested change. If there isn't any check for it currently, the condition should stay as is since it is least likely to interfere with existing behavior. cc @choumx

Reply:
I mean it depends on what the SSR API contract is. Is missing `init` or `init.status` exceptional? If so, we should add a `dev().error` for those cases to help surface bugs like "unexpected SSR template data".